Tablet layout home page

Eleein opened the pull request:
Create the home page layout for Tablet mode. This task was too big. Needed to be chunked into smaller parts

Eleein committed:
Add Footer component
 -Make Footer stick to bottom

Reported-by: Dania Duany

Eleein committed:
tablet-layout-home-page

Adjust the size and layout of the bottom text area for
a tablet-size device display.

Reported-by: Dania Duany

Eleein committed:
Refactor the default display to work with flex, because the later displays also
depend on flex, and this reduces the code footprint.

Eleein committed:
tablet-layout-home-page:
	-Add property for the Footer to stay at the bottom of page

Reported-By: Dania Duany

joe-crick commented on src/App.js:
```diff
@@ -2,19 +2,27 @@ import React from "react";
-          <div>Whatever, dude</div>
-        </Route>
-      </Switch>
+      <div className="content-wrap">
```
Don't call this class `content-wrap`. Come up with something more descriptive of what this element is doing. Also, why is this class a magic string, instead of a SCSS Module?

Eleein replied:
It didn't make sense to me to create a module for only one class. The idea is to have a wrapper holding the entire layout of the page.

joe-crick replied:
It's still best for you to use the SCSS modules. You keep things consistent that way. Otherwise, you're not taking the benefits of CSS Modules, and putting yourself at risk of having colliding CSS class names. Mixing normal class names with CSS Modules can increase the confusion of the code base, because of the mixture, and the fact that you're dropping the CSS Module protections you get in the places where you don't use it.

Eleein replied:
🆗

joe-crick commented on src/App.js:
```diff
@@ -2,19 +2,27 @@ import React from "react";
+            <Home />
+          </Route>
+          <Route path="*">
+            <div>Whatever, dude</div>
```
Avoid using potentially "interesting" comments like "Whatever, dude". More generic placeholders are safer 😄

joe-crick commented on src/Home/Home.module.scss:
```diff
@@ -39,9 +39,42 @@
+    background-position: center;
+  }
+  .description {
+    padding: 3vh 6vh 7vh;
```
Why are you using `vh` here? Are you trying to make the padding responsive?

Eleein replied:
As a matter of fact, I am :)

joe-crick replied:
👍

joe-crick commented on src/styles/global.scss:
```diff
@@ -141,3 +143,9 @@ body {
 }
+
+.App > div:first-child {
+ padding-bottom: 2.5rem;
```
Again, why `rem` here? Does this, too, need to be responsive?

Eleein replied:
yes, I believe it does

Eleein committed:
Open PR issues

	-Create theme folder
	-Define theme variables
	-change colors to hex format
	-change ZFooter name to Footer
	-Add module scss for App.js
Reported-by: Dania Duany

Eleein committed:
PR request
	-Change $greenSection to more descriptive $brand-greenSection
	-Change the theme variables to kebab case

Reported-by: Dania Duany

joe-crick commented on src/Home/Home.module.scss:
```diff
@@ -35,13 +34,45 @@
 .description {
-  background-color: #4f8d1f;
-  color: white;
+  background-color: $green-section;
```
Consider using a more descriptive name than `green-section`.

Eleein replied:
Thx

joe-crick commented on src/styles/theme.scss:
```diff
@@ -0,0 +1,20 @@
+$hero-bkgd-color: #fed631;
+$green-section: #4f8d1f;
+$white: #fff;
+$redStripes: #cb463e;
```
Make sure that your naming conventions are consistent. Choose between kebab case and camelCase.

Eleein replied:
Sure, thank you

Eleein committed:
PR changes
	-Change $descritpiveText to kebab case for styling purposes
Reported-By: Dania Duany

joe-crick commented on src/styles/theme.scss:
```diff
@@ -0,0 +1,20 @@
+$red-stripes: #cb463e;
+
+//font size
+$descriptiveText: 20px;
```
Should be kebab-case 😄

Eleein replied:
Sneaky one 🐌
